middlewares/auth.js

```diff
@@ -0,0 +1,18 @@
+const passport = require('passport');
+
+const auth = (req, res, next) => {
+  passport.authenticate('jwt', { session: false }, (err, user) => {
+    if (!user || err) {
```

Comment:
tam gdzie używasz aut nie musisz już sprawdzać czy user istnieje wywakać 401 i go pobierać bo od tego jest ten middlewere. Wyżej napisałem jak to powinno wyglądać bo teraz niby działa ale tam zawsze będzie ok jeśli tu nie wywali błędu.

2 sprawa tu trzeba sprawdzić jeszcze czy req.token ( - "Bearer " z przodu) jest taki sam jak user.token. Aktualnie jak się wyloguję to /current nadal mi działa (a raczej nie powinien).